client/sites/components/panel-sidebar/index.tsx

```diff
@@ -7,25 +7,19 @@ export function SidebarItem( { href, children }: { href: string; children: React
 	const isActive = window.location.pathname.startsWith( href );
 
 	return (
-		<li>
-			<Button
-				href={ href }
-				className={ clsx( 'panel-sidebar-tab', {
-					'panel-sidebar-tab--active': isActive,
-				} ) }
-			>
-				{ children }
-			</Button>
-		</li>
+		<Button
+			href={ href }
+			className={ clsx( 'panel-sidebar-tab', {
+				'panel-sidebar-tab--active': isActive,
+			} ) }
+		>
+			{ children }
+		</Button>
 	);
 }
 
 export function Sidebar( { children }: { children: ReactNode } ) {
-	return (
-		<div className="panel-sidebar">
-			<ul>{ children }</ul>
```

Comment:
I think the removal of ul-li breaks accessibility; I used to be able to iterate through the subtabs via my Tab key, similar to the first-level tab. Now, I can't

<img width="173" alt="image" src="https://github.com/user-attachments/assets/b2a8671d-8c0f-43a8-a36e-a1155d72ed5b">

Reply:
Ah, np I'll add it back.

Reply:
Not sure about the box-shadow color, I tried using grey but I think blue makes more sense, it's the color used on the rest of the form for focus as well.

Reply:
I think I am a bit confused, in the [initial implementation](https://github.com/Automattic/wp-calypso/pull/95585/), I didn't use ul/li but I can still use Tab key. Not sure why. It looks like the "introduction" of ul/li here adds an unnecessary left-margin. Sorry for the confusion!

In any case, the expected end-user behavior here is:

- I can use Tab to loop through the subtabs
- No left margin

I think using ul/li is the best practice here still, because it is friendly to accessibility reader? Not really sure. 😄